torch/fx/passes/reinplace.py

```diff
@@ -252,6 +253,7 @@ def matching_view_metadata(a, b):
         assert isinstance(base.meta["fake_result"], FakeTensor)
         assert isinstance(mutated_view, Node)
         assert isinstance(mutated_view.meta["fake_result"], FakeTensor)
+        assert not isinstance(n.target, str)
```

Comment:
Tiny quibble: I personally prefer positive assertions for types so it's easier to understand what `n.target` is supposed to be. Also, from having to debug these assertions going sour, I tend to put something to print in the second parameter, in the most extreme case it would go like this:

`assert isinstance(n.target, (foo, bar, baz)), (n.target, type(n.target))`

but most often

`assert isinstance(n.target, (foo, bar, baz)), n.target`

Reply:
@rec This one is actually necessarily a negative assertion, because the other possibility is `Callable[..., Any]`, which can't be directly asserted for as a generic type.